packages/webdriverio/src/index.ts

```diff
@@ -37,9 +39,16 @@ export const remote = async function(
     params: RemoteOptions,
     remoteModifier?: (client: WebDriverTypes.Client, options: Options.WebdriverIO) => WebDriverTypes.Client
 ): Promise<WebdriverIO.Browser> {
-    logger.setLogLevelsConfig(params.logLevels as any, params.logLevel)
     const keysToKeep = Object.keys(process.env.WDIO_WORKER_ID ? params : DEFAULTS) as (keyof RemoteOptions)[]
     const config = validateConfig<RemoteOptions>(WDIO_DEFAULTS, params, keysToKeep)
+
+    if (config.outputDir) {
```

Comment:
We already do this in other places. Instead of adding the logic here, I think we might need to move it from another location this this. Curious to hear your thoughts on this Christian as I feel like we are duplicating this code in a few places now (launcher, worker, the file in this PR and potentially more)

Reply:
I agree.

I suggest to create a helper function in `@wdio/utils` like this:

```ts
export function enableFileLogging (outputDir?: string) {
    if (!outputDir) {
        return
    }

    await fs.mkdir(path.join(config.outputDir), { recursive: true })
    process.env.WDIO_LOG_PATH = path.join(config.outputDir, 'wdio.log')
}
```

and import this in:

- `https://github.com/webdriverio/webdriverio/blob/fa5f02f24b64372d2791814bde1306788319e30e/packages/wdio-cli/src/launcher.ts#L85-L88`
- `https://github.com/webdriverio/webdriverio/blob/fa5f02f24b64372d2791814bde1306788319e30e/packages/wdio-runner/src/utils.ts#L63-L65`
- and this place

Does that make sense?

Reply:
For me it does with the exception we might need to add the check of the variable already exists or not. Not sure if that is needed anymore now though. Thank you for your feedback Christian 😉

Reply:
Thanks for your feedback @erwinheitzman @christian-bromann, I'll take it along in my next commit.

Reply:
Hi @erwinheitzman , @christian-bromann,
I updated this pull request following what we've discussed. Please help take a look. Thanks!